src/OrleansRuntime/Cancellation/CancellationSourcesExtension.cs

```diff
@@ -37,29 +37,12 @@ public Task CancelTokenSource(GrainCancellationToken token)
             return gct.Cancel();
         }
 
-        internal GrainCancellationToken GetOrCreateCancellationToken(GrainCancellationToken token)
-        {
-            GrainCancellationToken localToken;
-            if (_cancellationTokens.TryFind(token.Id, out localToken))
-            {
-                if (token.IsCancellationRequested && !localToken.IsCancellationRequested)
-                {
-                    localToken.Cancel();
```

Comment:
Missed this one: initially Cancel wasn't async. And after redesign this call became unnecessary.